src/tests/system/tests/test_ipa.py

```diff
@@ -0,0 +1,88 @@
+"""
+SSSD IPA tests.
+
+:requirement: ipa
+"""
+
+from __future__ import annotations
+
+import time
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.ipa import IPA
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.ticket(bz=1926622)
+@pytest.mark.topology(KnownTopology.IPA)
+def test_sssd_ipa_indicators(client: Client, ipa: IPA):
+    """
+    :title: Add support to verify authentication
+     indicators in pam_sss_gss
+    :setup:
+        1. Add pam_sss_gss configuration to /etc/sssd/sssd.conf
+        2. Add pam_sss_gss.so to /etc/pam.d/sudo
+        3. Restart SSSD
+        4. Enable SSSD debug logs
+    :steps:
+        1. Switch to 'admin' user
+        2. obtain Kerberos ticket.
+        3. Create sudo configuration that allows an admin to
+           run SUDO rules
+        4. Try 'sudo -l' as admin
+        5. As root, check content of sssd_pam.log
+       `6. Check if acquired service ticket has req. indicators: 0
+       `7. Add pam_sss_gss configuration to /etc/sssd/sssd.conf
+       `8. Check if acquired service ticket has req.
+           indicators: 2
+    :expectedresults:
+        1. Should succeed
+        2. Should succeed
+        3. Should succeed
+        4. Should succeed
+        5. Should succeed
+        6. Should succeed
+        7. Should succeed
+        8. Should succeed
+    :customerscenario: True
+    """
+    user = "admin"
+    test_password = "Secret123"
+
+    client.sssd.config["pam"] = {
+        "pam_gssapi_services": "sudo, sudo-i",
+        "pam_gssapi_indicators_map": "hardened, sudo:pkinit, sudo-i:otp",
+    }
+    client.sssd.start()
+    for path in ["/etc/pam.d/sudo", "/etc/pam.d/sudo-i"]:
```

Comment:
We want to avoid things like this, especially pam, you maybe able to use authselect utils to do it, it should be noted that the framework needs to support this.

Reply:
client.fs.sed is a module that was added recently 

https://github.com/next-actions/pytest-mh/pull/62

I have no idea what kind of support you are looking for ?? You can create a separate ticket where we can discuss this .